### DISCUSSION TITLE
我有生之年到底还能不能考过

### DISCUSSION THREAD

**Discussion by zeorzero15** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#15] [General]
出这个题和制定这个解锁方案的人，我用全人类，所有人的背痛和厄运诅咒他

**Comment by MlgmXyysd** [comment]
![Screenshot_2023-12-14-15-22-40-28_948cd9899890cbd5c2798760b2b95377.jpg](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/aa32100d-3421-4c81-915e-696f57689439)

那你估计要寄了，现在申请审批还要政审（笑）
